group-community.md

```diff
@@ -17,9 +17,9 @@ _Well, now, as i'm writing this, i've [submitted](https://lov.linkeddata.es/data
 
 I can offer people a place to stay. But how do they ever find me? At the moment, our data live in different places, and we have no way to find each other. Internet is a vast place...
 
-### 1. Crawl
+### 1. Crawl a network of friends of a friends
 
-Well, people can look at their friends, and friends of friends, and friends of friends of friends, and so on, and see if somebody offers a place to stay; and where it is.
+Well, people can ask (look at the profile of) their friends, and friends of friends, and friends of friends of friends, and so on, and see if somebody offers a place to stay; and where it is.
```

Comment:
`people can look at the profile of their friends` or `people can look at the data of their friends` sounds good to me. To me `ask` represents act of asking them, and getting answers, which they don't do.